lagom-eventstore-example/resources/lagom-event-store-example-greetings.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lagom-Project EventStore example\n",
+    "\n",
+    "This notebook is part of the Lagom-event-store-example application. The Lagom code generates data that is stored into IBM EventStore and then you may read and analyse the data using this notebook. See the repository [README.md](https://github.com/lagom/ibm-integration-examples/tree/master/lagom-eventstore-example) for more details.\n",
+    "\n",
+    "This notebook uses the Scala API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+       "## Table of contents\n",
+       "1. [Setup](#setup)<br>\n",
+       "2. [Query the table](#query-table)<br>\n",
+       "   2.1 [Connect to Project EventStore](#connect-to-es-two)<br>\n",
+       "   2.2 [Create sqlContext using EventSession](#create-sqlContext)<br>\n",
+       "   2.3 [Prepare a DataFrame for the query](#prepare-DataFrame)<br>\n",
+       "   2.4 [Run the SQL query](#run-query)<br>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"setup\"></a>\n",
+    "## 1. Setup"
+   ]
+  },
+    {
+     "cell_type": "code",
+     "execution_count": 2,
+     "metadata": {
+      "collapsed": false
+     },
+     "outputs": [],
+     "source": [
+      "   // This settings must match the values you configured in 'ibm-event-store.conf'.\n",
+      "   \n",
+      "   // NOTE: The database name is case sensitive.\n",
+      "   // example: val DATABASE_NAME: String = \"greetingsDB\"\n",
+      "   val DATABASE_NAME: String = \"greetingsDB\"\n",
```

Comment:
I changed the case in the config file to match the notebook in 799c04de35cb13912458e63f7c2bb3b0b7bc065b, and then the case in the notebook was changed later so they don't match again :smile: You can drop my commit from the branch.

Reply:
I'll go ahead and rebase the branch without that commit. Then I'll merge.